handlers/devicetype.go

```diff
@@ -21,6 +21,21 @@ func NewDeviceTypeHandler(service ports.DeviceTypeService) *DeviceTypeHandler {
 	}
 }
 
+// Handle API endpoint for listing all device types.
+func (h *DeviceTypeHandler) List(w http.ResponseWriter, r *http.Request) error {
```

Comment:
Would be more consistent if this was called GetAll, like the underlying function being called.